draft-ietf-httpbis-http2bis.xml

```diff
@@ -1415,6 +1415,16 @@
         <t>
           A list of error codes is included in <xref target="ErrorCodes"/>.
         </t>
+        <t>
+          It is possible that an endpoint will encounter frames that would cause multiple errors. Implementations MAY process
+          multiple errors triggered by a single frame in any order, within the limitations set out in this section. Once an
```

Comment:
```suggestion
          It is possible that an endpoint will encounter frames that would cause multiple errors. Implementations MAY discover
          multiple errors during processing, but they SHOULD report at most one stream and one connection error as a result.
```

And then I think that we need a new paragraph, though you can join the text I suggest below.